.travis.yml

```diff
@@ -10,10 +10,17 @@ addons:
     - liblapack-dev
     - python-numpy
     - swig
+before_script:
+  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.14-Linux-x86_64.sh
```

Comment:
Miniconda isn't necessary, but I think it's a good idea to make sure we're compatible with it, and we'll need it eventually when we have a conda package.  Plus, it gives us flexibility to test against any package versions we want using lightweight conda environments.  Also, it looks like the default pythons on Ubuntu 12.04 (which is what Travis uses) are 2.7.3 and 3.2.3, so Miniconda is an easy way to test more recent versions.